docker-java/src/test/java/com/github/dockerjava/cmd/ListImagesCmdIT.java

```diff
@@ -54,6 +57,28 @@ public void listImagesWithDanglingFilter() throws DockerException {
         assertTrue(imageInFilteredList);
     }
 
+    @Test
+    public void listImagesWithReferenceFilter() throws DockerException {
+        String tag = "" + RandomUtils.nextInt(0, Integer.MAX_VALUE);
+
+        dockerRule.getClient().tagImageCmd("busybox:latest", "docker-java/busybox", tag).exec();
+        List<Image> images = dockerRule.getClient().listImagesCmd().withReferenceFilter("docker-java/busybox")
+            .exec();
+        assertThat(images, hasSize(1));
+        dockerRule.getClient().removeImageCmd("docker-java/busybox:" + tag).exec();
```

Comment:
Should be in `finally` block here and `listImagesWithFilter`

Reply:
this one should too, right? https://github.com/docker-java/docker-java/blob/8e77a69467814c8500e5a360bfce46602ca841fb/docker-java/src/test/java/com/github/dockerjava/cmd/TagImageCmdIT.java#L17-L19